stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -137,6 +137,7 @@ function versioninfo(io::IO=stdout; verbose::Bool=false)
     println(io, "  WORD_SIZE: ", Sys.WORD_SIZE)
     println(io, "  LIBM: ",Base.libm_name)
     println(io, "  LLVM: libLLVM-",Base.libllvm_version," (", Sys.JIT, ", ", Sys.CPU_NAME, ")")
+    println(io, "  Threads.nthreads(): ", Base.Threads.nthreads())
```

Comment:
```suggestion
    println(io, "  nthreads: ", Threads.nthreads(), " on ", Sys.CPU_THREADS, " cores")
```

Reply:
Should we say "virtual cores" instead of just "cores", to distinguish from physical cores?

Reply:
Done! I used the text "virtual cores", but we can change it to "cores" if desired.